### DISCUSSION TITLE
Async Webserver

### DISCUSSION THREAD

**Discussion by ZinnerC** [earlephilhower/arduino-pico#1972] [Q&A]
I'm currently checking out if there is a async server available for the RP2040 via LWIP (W5100).  Haven't had luck so far. Might be that somebody from you found something which I could give a try.
Thanks 😁

**Comment by ZinnerC** [comment]
I found something but as far as I can see this only works for the Pico W with CYW43439.
https://github.com/khoih-prog/AsyncWebServer_RP2040W
Also this library is archived. So I assume this is not maintained anymore.

Does anybody know about a derived project?
Or how could this be used in work with LWIP and a W5100 for Lan connection.

**Reply by earlephilhower** [reply]
I suggest commenting out that check for the PICO_W define and giving it a go.  All native networking on the Pico in this core is through LWIP, including the CYW43, ESP32, W5x00 and ENC68J.
